use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -0,0 +1,491 @@
+    + [Install necessary bundles for Clear Linux and disable kernel updates](#install-necessary-bundles-for-clear-linux-and-disable-kernel-updates)
+    + [Replace Clear Linux kernel with LTS kernel](#replace-clear-linux-kernel-with-lts-kernel)
+    + [Allow insecure modules to load to host](#allow-insecure-modules-to-load-to-host)
+    + [Download Intel QAT drivers](#download-intel-qat-drivers)
+    + [Compile Intel QAT drivers against host kernel](#compile-intel-qat-drivers-against-host-kernel)
+    + [Copy Intel QAT configuration files and enable VF’s](#copy-intel-qat-configuration-files-and-enable-vf-s)
+    + [Expose and Bind Intel QAT virtual functions to VFIO-PCI (Every reboot)](#expose-and-bind-intel-qat-virtual-functions-to-VFIO-pci--every-reboot-)
+    + [Check Intel QAT virtual functions are enabled](#check-intel-qat-virtual-functions-are-enabled)
+  * [Prepare Kata Containers](#prepare-kata-containers)
+    + [Download Kata Kernel Source](#download-kata-kernel-source)
+    + [Build Kata Kernel](#build-kata-kernel)
+    + [Copy Kata Kernel](#copy-kata-kernel)
+    + [Prepare Kata root filesystem](#prepare-kata-root-filesystem)
+    + [Compile Intel QAT drivers for Kata Containers Kernel and add to Kata Containers rootfs](#compile-intel-qat-drivers-for-kata-containers-kernel-and-add-to-kata-containers-rootfs)
+    + [Copy Kata rootfs](#copy-kata-rootfs)
+    + [Update Kata configuration.toml to point to custom kernel and rootfs](#update-kata-configurationtoml-to-point-to-custom-kernel-and-rootfs)
+  * [Verify Intel QAT works in a Docker Kata Containers container](#verify-intel-qat-works-in-a-docker-kata-containers-container)
+  * [Build OpenSSL Intel QAT engine container](#build-openssl-intel-qat-engine-container)
+    + [Test Intel QAT in Docker](#test-intel-qat-in-docker)
+    + [Troubleshooting](#troubleshooting)
+  * [Optional Scripts](#optional-scripts)
+    + [Verify Intel QAT card counters are incremented](#verify-intel-qat-card-counters-are-incremented)
+    + [Unbind all VF’s from VFIO driver, reset, and rebind to the host](#unbind-all-vf-s-from-VFIO-driver--reset--and-rebind-to-the-host)
+
+# Introduction
+
+Intel\* QuickAssist Technology (Intel\* QAT) provides hardware acceleration 
+for security (cryptography) and compression. These instructions cover the 
+steps for [Clear Linux](https://clearlinux.org/)\* but can be adapted to any 
+Linux\* distribution. Your distribution may already have the Intel QAT 
+drivers, but it is likely they are do not contain the necessary user space 
+components. These instructions guide the user on how to download the kernel 
+sources, compile kernel driver modules against those sources, and load them 
+onto the host as well as preparing a specially built Kata Containers® kernel 
+and custom Kata Containers rootfs.  
+
+## Helpful Links before starting
+
+[Intel QAT Engine](https://github.com/intel/QAT_Engine)
+
+[Intel QuickAssist Technology at 01.org](https://01.org/
```

Comment:
I think the line breaks in these links make them broken (not rendering as links). Hmm, did you flow your markdown to 80 columns perchance ?

Reply:
I am not sure what is wrong. They work for me when I use sublime and a plugin to render markdown.  However, I am trying this on an updated version of this md file after I ran it through dos2unix.  I don't have word wrap turned on and have manually did a break at around 78 columns.  I should note that 01.org is not working for me or another coworker but I assume it will be fixed at some point.

Reply:
Links still look broken - still probably been line-break'd by your editor, which does not work for link URLs.

Reply:
Hi @eadamsintel - thanks for the updates. But these links are *still* broken. Please edit the document to "join" the two lines back into one. The doc shows for example:

```
[Intel QuickAssist Technology at 01.org](https://01.org/
intel-quickassist-technology)
```

That's invalid - you can see this by clicking on the "view file" preview - it doesn't render as a link correctly. These two lines need to be joined together onto a single line to become:

```
[Intel QuickAssist Technology at 01.org](https://01.org/intel-quickassist-technology)
```

It may be that your editor is "too clever" and somehow handles such broken links, but I'm afraid nothing else does ;)

If you are using Windows, for speed, you could even edit the file in MS notepad to re-join these broken lines (as long as you remember to `dos2unix` the file before re-pushing here).

Reply:
I think the problem was how the editor is setup.  I changed some settings and see the broken link. better now and have joined it. I'll fix and resubmit shortly once I go through the rest of the feedback.